components/federal_banner.js

```diff
@@ -40,52 +23,80 @@ const container = css`
 
   .svg-container {
     /* same as the width value in our svg */
-    width: 400px;
+    width: 350px;
     height: 30px;
   }
 
   .svg-logo {
-    width: 400px;
+    width: 350px;
     fill: white;
   }
 
-  ${mediaQuery.xs(css`
+  @media only screen and (max-width: ${globalTheme.max.xs}) {
     .svg-container {
       width: 220px;
       height: 30px;
-
       svg {
         width: 220px;
         height: 30px;
         fill: white;
       }
     }
-  `)};
+  }
+`;
+
+const flex = css`
+  width: 100%;
+  height: 100%;
+  border-top: 1px solid ${globalTheme.colour.navy};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  @media only screen and (max-width: ${globalTheme.max.sm}) {
+    height: 52px;
+  }
+`;
+
+const flexItems = css`
+  flex: 1;
+  width: 100%;
+  text-align: right;
+  @media only screen and (max-width: ${globalTheme.max.sm}) {
+    text-align: left;
+  }
 `;
 
 class FederalBanner extends Component {
   render() {
-    const { t, skipLink, i18n } = this.props;
+    const { t, skipLink, i18n, url } = this.props;
     return (
-      <div>
-        <SkipToMainContent skipLink={skipLink} t={t} />
-        <div className={container}>
-          <div className="svg-container">
-            <FIP fillColor="white" t={t} />
+      <Grid container>
+        <Grid item xs={12} md={6}>
+          <SkipToMainContent skipLink={skipLink} t={t} />
```

Comment:
Just a note for the future PR: getting the styling just right to match the VAC site header with the breakpoints might be a bit tricky to do anyway, because of the space the SkipLink takes up.